io.greenfire.Greenery.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.greenfire.Greenery.desktop</id>
```

Comment:
```suggestion
  <id>io.greenfire.Greenery</id>
```

never `.desktop` in app ids

Reply:
This has been removed, I am working on this update and the others.  Should I re-submit to this same PR?